src/ast.rs

```diff
@@ -284,6 +290,24 @@ pub enum LetStatement {
     },
 }
 
+// 12. Control statements
+
+#[derive(Debug, PartialEq, Eq)]
+pub enum Relation {
+    LessThan,
+    LessThanOrEqualTo,
+    EqualTo,
+    GreaterThan,
+    GreaterThanOrEqualTo,
+    NotEqualTo,
```

Comment:
How is this related to `EqualityRelation::NotEqualTo`?

Reply:
@jeschkies it's like this in the spec, I know it sounds weird. I'll double check.